test/verify/check-machines

```diff
@@ -1622,7 +1622,7 @@ class TestMachines(MachineCase, StorageHelpers, NetworkHelpers):
         # We don't want to use start_vm == False because if we get a separate install phase
         # virt-install will overwrite our changes.
 
-        if self.machine.image not in ["fedora-33", "fedora-testing"]:
+        if self.machine.image not in ["fedora-33", "fedora-testing", "debian-testing"]:
```

Comment:
BTW, next time we touch that we should invert the list -- keep an explicit list of OSes that have it broken, so that we can *shrink* the list to zero instead of expanding it.

Thanks!